python/lsst/cp/pipe/utils.py

```diff
@@ -784,6 +785,94 @@ def ddict2dict(d):
     return dict(d)
 
 
+class Pol2D:
```

Comment:
Can we use the afw/math polynomial code here instead?  Adding a duplicate class seems like a bad idea.

Reply:
I filed a ticket: DM-42196